File: src/test/java/net/dinkla/raytracer/samplers/PureRandomTest.java
package net.dinkla.raytracer.samplers;

import org.junit.jupiter.api.BeforeEach;

public class PureRandomTest extends GeneratorTest {

    @BeforeEach
    public void initialize() {
        new PureRandom().generateSamples(NUM_SAMPLES, NUM_SETS , samples);
    }

}

File: src/test/kotlin/net/dinkla/raytracer/samplers/PureRandomTest.kt
package net.dinkla.raytracer.samplers

import org.junit.jupiter.api.BeforeEach

class PureRandomTest : AbstractGeneratorTest() {

    @BeforeEach
    override fun initialize() {
        PureRandom().generateSamples(NUM_SAMPLES, NUM_SETS, samples)
    }

}
